portal_partner_select_all/__openerp__.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 # Copyright 2018 Tecnativa S.L. - David Vidal
+# Copyright 2019 Coop IT Easy SCRL fs - Robin keunen
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Portal Partner Select All',
-    'version': '10.0.1.0.1',
+    'version': '9.0.1.0.1',
     'category': 'Custom',
     'author': 'Tecnativa,'
+              'Coop IT Easy,'
```

Comment:
Sorry, but this kind of simple migrations, as welcome as they ar, can't entitle your work as author.